server/nimble_server.py

```diff
@@ -111,6 +144,16 @@ def cqgi_model_script(file_name, params):
     return res
 
 
+# http://127.0.0.1:8000/wakoma/nimble/components
+@app.get("/wakoma/nimble/components")
+async def get_body(request: Request):
+    # Load the devices.json file
+    with open('../devices.json', 'r') as component_file:
```

Comment:
Best practice is to explicitly specify encoding with `open('../devices.json', 'r', encoding='utf-8')`